### ISSUE TITLE
Build warning seen while building boost/status

### ISSUE THREAD

**Issue by jeking3** [boostorg/geometry#524]
toolset=clang-6 cxxstd=03 variant=release
```
../boost/geometry/srs/projections/proj/isea.hpp:1070:25: warning: enumeration values 'isea_addr_geo', 'isea_addr_interleave', and 'isea_addr_plane' not handled in switch [-Wswitch]
                switch (g->output) {
                        ^
1 warning generated.
```

**Comment by awulkiew** [comment]
Thanks, I'll try to push a fix to 1.69.

**Comment by awulkiew** [comment]
Done
